app/sliders/importer.py

```diff
@@ -167,59 +179,46 @@ def apply_mapping(self, row):
             return mapping_employees_n.get(row["Nr. der Klas-\nsifikation"], row["Wirtschaftsgliederung"])
         else:
             return row["Wirtschaftsgliederung"]
-
-    def init_nested_dict(self):
-        """
-        Helper function.
-        Initialize a default dictionary recursively,
-        i.e., a nested dictionary with an arbitrary number of levels
-        and where keys are created automatically if missing.
-
-        :return: defaultdict, a nested dictionary
-        """
-        return defaultdict(self.init_nested_dict)
     
 
 class FUEDataParser:
     def parse(self, sheets):
+        self.config = self.get_config('donut_fue')
         data = self.extract(sheets)
 
         return data
     
+    def get_config(self, chart_name):
```

Comment:
This function can go out of the class, so we can reuse it for different parsers, like you did with `init_nested_dict()`

Reply:
👌